solutions/usaco-574.mdx

```diff
@@ -107,5 +106,37 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+from collections import deque
+	
+with open('feast.in') as r:
+	T, A, B = map(int, r.readline().split())
+
+fullness, queue = set(), deque()
+fullness.add((0, 0))
+queue.append((0, 0))
+
+while queue:
+	node = queue.pop()
+	curr, not_full = node[0], node[1]
+	for fruit in (A, B):
+		if not not_full:
+		# drink water
+		water = (curr // 2, 1 - not_full)
```

Comment:
?

Reply:
yeah.
<img width="885" alt="Screen Shot 2022-05-20 at 8 24 15 AM" src="https://user-images.githubusercontent.com/52589420/169424884-d9b48307-047c-43f0-8378-6cbc21593b4f.png">

Reply:
i ask this because this is literally an indentation error

Reply:
oh lol sorry i put tabs after